staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -2269,3 +2261,299 @@ func TestLoadObjects(t *testing.T) {
 		})
 	}
 }
+
+func TestApplyObjects(t *testing.T) {
+	testdirs := []string{"testdata/prune/simple"}
```

Comment:
I noticed while working on my PR that this test suite does already have its own directory, but it is rooted higher up in the tree: `staging/src/k8s.io/kubectl/testdata/apply`. Perhaps we should use that?

Reply:
The convention I like to stick to is that the `testdata` should live in the same directory as the tests.  Sometimes it has to be moved higher up if they are shared, but that is often a code smell (e.g. false sharing)